tests/test_storage.py

```diff
@@ -38,20 +39,20 @@ def test_reindex():
     engine = Elasticsearch()
     storage = Storage(engine)
     storage.delete()
-    storage.create(
-            'reindexing',
-            [('articles', schema)])
+    storage.create('reindexing', schema)
     
     # Write data
-    list(storage.write('reindexing', 'articles', datas, schema['primaryKey']))
-    assert sorted(list(storage.read('reindexing', doc_type='articles')),
-                  key=lambda x:x['id']) == datas
+    list(storage.write('reindexing', datas, schema['primaryKey']))
+
+    engine.indices.flush('_all')
+    time.sleep(3)
```

Comment:
Not sure why you sleep after flushes here and elsewhere. Is it something we can avoid? I know it is only the test

Reply:
Turns out that if reading immediately after writing, some results are not returned yet (as ES is eventually consistent).
To ensure that we read what we've just written I added this short sleep (a shorter one would also probably suffice but I wanted to be on the safe side).